pkg/volume/csi/csi_test.go

```diff
@@ -293,6 +293,8 @@ func TestCSI_VolumeAll(t *testing.T) {
 			}
 
 			var devicePath string
+			waitForAttachDone := make(chan string)
+			defer close(waitForAttachDone)
```

Comment:
Hey, It 's not suitable here to call defer. Usually we put defer on outside, in main process.  But here the variable waitForAttachDone is in the for loop(line 222). I understand, waitForAttachDone will be assign in every loop. So this code mean it will set every time. It 's very easy to case Ambiguity.